File: Program's_Contributed_By_Contributors/Python_Programs/linear_search.py
def linear_search(arr, n, x):
    for i in range(0, n):
        if (arr[i] == x):
            return i
    return -1


## Testing
arr = [2, 4, 6, 8, 3, 6, 9]
result = linear_search(arr, len(arr), 3)
if(result == -1):
    print("Element is not present in array")
else:
    print(f"Element is present at index: {result}")

File: java programs/LinearSearch.java
class LinearSearch {
    // This function returns index of element x in arr[]
    static int search(int arr[], int n, int x)
    {
        for (int i = 0; i < n; i++) {
            // Return the index of the element if the element
            // is found
            if (arr[i] == x)
                return i;
        }
 
        // return -1 if the element is not found
        return -1;
    }
 
    public static void main(String[] args)
    {
        int[] arr = { 3, 4, 1, 7, 5 };
        int n = arr.length;
         
        int x = 4;
 
        int index = search(arr, n, x);
        if (index == -1)
            System.out.println("Element is not present in the array");
        else
            System.out.println("Element found at position " + index);
    }
}